SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -46,9 +47,14 @@ void USpatialPlayerSpawner::ReceivePlayerSpawnRequest(Schema_Object* Payload, co
 		UniqueIdReader << UniqueId;
 
 		FName OnlinePlatformName = FName(*GetStringFromSchema(Payload, 3));
+		bool bSimulatedPlayer = Schema_GetBool(Payload, 4);
 
 		URLString.Append(TEXT("?workerAttribute=")).Append(Attributes);
-
+		if (bSimulatedPlayer)
+		{
+			URLString += TEXT("?simulatedPlayer=1");
```

Comment:
Couldn't `simulatedPlayer=1` be attached to the URL in the `SpawnPlayerRequest` instead of needing to have a separate bool?

Reply:
Isn't this cheaper?